fiftyone/core/map/batcher/slice_batch.py

```diff
@@ -0,0 +1,56 @@
+"""
+Abstract mapping backend
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+from typing import List, Optional, TypeVar
+
+import numpy as np
+
+import fiftyone.core.map.batcher.batch as fomb
+from fiftyone.core.map.typing import SampleCollection
+
+T = TypeVar("T")
+
+
+class SampleSliceBatch(fomb.SampleBatch):
+    """Sample batch using slices"""
+
+    @classmethod
+    def split(
+        cls,
+        sample_collection: SampleCollection[T],
```

Comment:
Is this different from a fiftyone e sample collection? You can slice a dataset and dataset view, but I don't know if you can actually slice a sample collection

Reply:
sample collection usually denotes either-dataset-or-view.